include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -2085,21 +2085,30 @@ namespace internal
           shape_info.dofs_per_component_on_cell;
         constexpr unsigned int hess_dim = dim * (dim + 1) / 2;
 
-        AlignedVector<VectorizedDouble> cell_points(dim * n_mapping_points);
-        AlignedVector<VectorizedDouble> face_quads(dim * n_q_points);
-        AlignedVector<VectorizedDouble> face_grads(dim * dim * n_q_points);
-        AlignedVector<VectorizedDouble> face_grad_grads(dim * hess_dim *
-                                                        n_q_points);
+        MappingInfoStorage<dim - 1, dim, double, VectorizedDouble> temp_data;
+        temp_data.copy_descriptor(my_data);
+        FEEvaluationBaseData<dim, double, true, VectorizedDouble> eval_int(
+          {&shape_info, nullptr, &temp_data, 0, 0}, 0, true, 0);
+        FEEvaluationBaseData<dim, double, true, VectorizedDouble> eval_ext(
+          {&shape_info, nullptr, &temp_data, 0, 0}, 0, false, 0);
 
-        AlignedVector<VectorizedDouble> scratch_data(
-          dim * (2 * n_q_points + 3 * n_mapping_points));
+        AlignedVector<VectorizedDouble> evaluation_data, evaluation_data_ext;
+        eval_int.set_data_pointers(&evaluation_data, dim);
+        eval_ext.set_data_pointers(&evaluation_data_ext, dim);
 
         for (unsigned int face = begin_face; face < end_face; ++face)
           for (unsigned vv = 0; vv < n_lanes; vv += n_lanes_d)
             {
+              internal::MatrixFreeFunctions::FaceToCellTopology<
+                VectorizedDouble::size()>
+                face_double;
+              face_double.interior_face_no = faces[face].interior_face_no;
+              face_double.exterior_face_no = faces[face].exterior_face_no;
+              face_double.face_orientation = faces[face].face_orientation;
+              face_double.subface_index    = faces[face].subface_index;
```

Comment:
Copy constructor?

Reply:
We can't because we have different array lengths due to different `VectorizedArrayType`s, so the assignment is ambiguous.